diff --git a/src/vmm/src/lib.rs b/src/vmm/src/lib.rs
index 88f94ca..53e1379 100644
--- a/src/vmm/src/lib.rs
+++ b/src/vmm/src/lib.rs
@@ -947,7 +947,7 @@ impl Vmm {
         for handle in self.vcpus_handles.iter() {
             match handle
                 .response_receiver()
-                .recv_timeout(Duration::from_millis(100))
+                .recv_timeout(Duration::from_millis(1000))
             {
                 Ok(VcpuResponse::Resumed) => (),
                 _ => return Err(StartMicrovmError::VcpuResume),
